angular.json

```diff
@@ -1,5 +1,8 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "cli": {
+    "analytics": false
+  },
```

Comment:
Remove changes from package-lock.json. file. They are not supposed to be in your PR!

Reply:
fixed